rs/ic_os/config/src/main.rs

```diff
@@ -116,7 +116,7 @@ pub fn main() -> Result<()> {
                     );
                     mgmt_mac
                 }
-                None => get_ipmi_mac()?,
+                None => get_mgmt_mac()?,
```

Comment:
I like the idea of pushing this branch into a function, could you please use the bypass method that we already have, instead?

You can see this just a few lines above your changes here:
https://github.com/dfinity/ic/blob/bc2c84c5e3dc14bedb7ef9bbe231748886a34a29/rs/ic_os/config/src/main.rs#L110-L120

Reply:
I have moved the code that decides which MAC to use to the mac_address.rs file.